src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/DBFilterBase.java

```diff
@@ -112,12 +114,12 @@ public BuiltCondition get() {
           return BuiltCondition.of(
               DATA_CONTAINS,
               Predicate.CONTAINS,
-              getGrpcValue(getHashValue(new DocValueHasher(), key, value)));
+              new JsonTerm(getHashValue(new DocValueHasher(), key, value)));
         case MAP_EQUALS:
           return BuiltCondition.of(
-              BuiltCondition.LHS.mapAccess(columnName, Values.of(key)),
+              BuiltCondition.LHS.mapAccess(columnName, Values.NULL),
```

Comment:
Why null? This seems odd.

Reply:
Stargate api query builder needs a value to be passed but it's not used. Instead serializing the data predefined  NULL value is passed.